lasy/profiles/longitudinal/cosine_profile.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+
+from .longitudinal_profile import LongitudinalProfile
+
+
+class CosineLongitudinalProfile(LongitudinalProfile):
+    r"""
+    Derived class for the analytic longitudinal truncated cosine profile of a laser pulse.
+
+    More precisely, the longitudinal envelope
+    (to be used in the :class:CombinedLongitudinalTransverseProfile class)
+    corresponds to:
+
+    .. math::
+
+        \mathcal{L}(t) = \cos\left({ \frac{\pi}{2} \frac{t-t_{peak}}{\tau_{fwhm}} }\right)
+            \theta\left({ \frac{t-t_{peak}}{\tau_{fwhm}} + 1 }\right)
+            \theta\left({ 1 - \frac{t-t_{peak}}{\tau_{fwhm}}} \right)
+            \exp\left({ + i (\phi_{cep} + \omega_0 t_{peak} ) }\right)
+
+
+    Parameters
+    ----------
+    tau_fwhm : float (in second)
+        The Full-Width-Half-Maximum duration of the intensity distribution of the pulse,
```

Comment:
the description of the `wavelength` parameter is missing here.

Reply:
You are right! Thanks for having spotted this!